### PR TITLE
DEVELOPER-1226 - Publish Java EE 7 Samples as JBoss Developer Demos

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

File: javaee7-samples/concurrency.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: Java EE 7 Samples: Concurrency Utilities
+technologies:
+  - Java EE 7
+  - Concurrency Utilities
+summary: The JSR 346 updates and clarifications to CDI 1.0 along much requested features.
```
Comment (paulrobinson) [inline javaee7-samples/concurrency.yaml]: Copy and paste error?

File: javaee7-samples/jaspic.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+title: Java EE 7 Samples: JASPIC - Java Authentication Service Provider Interface for Containers
+technologies:
+  - Java EE 7
+  - Javamail
```
Comment (paulrobinson) [inline javaee7-samples/jaspic.yaml]: is javamail a related technology here?

**Comment by paulrobinson** [comment]
Other than my comments, this looks good.

**Comment by rafabene** [comment]
Indeed! There was copy/paste errors. Thanks for catching it. Rebased and fixed.

**Pull request merged**